linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive/rule.yml

```diff
@@ -45,7 +56,11 @@ references:
 ocil_clause: 'it is commented out or not configured properly'
 
 ocil: |-
-    To ensure the SSH idle timeout will occur when the <tt>ClientAliveInterval</tt> is set, run the following command:
+    To ensure <tt>ClientAliveInterval</tt> is set correctly, run the following command:
     <pre>$ sudo grep ClientAliveCountMax /etc/ssh/sshd_config</pre>
-    If properly configured, output should be:
+    If properly configured, the output should be:
     <pre>ClientAliveCountMax {{{ xccdf_value("var_sshd_set_keepalive") }}}</pre>
+    If the option is set to <tt>0</tt>, then the SSH idle timeout occurs precisely when
+    the <tt>ClientAliveInterval</tt> is set.
+    If the option is set to a number greater than <tt>0</tt>, then the idle session will be disconnected after
+    <tt>ClientAliveInterval * ClientAliveCountMax</tt> seconds.
```

Comment:
If I understood correctly what the SSH SME said in the [bugzilla description](https://bugzilla.redhat.com/show_bug.cgi?id=1873547), this statement if completely false since it will NOT end the session just by the user being idle, but when there is a connectivity issue in the TCP connection. Below are the statements provided by the SME extracted from the bugzilla description:

`What the ssh protocol does have is the ability to monitor the TCP connection to ensure the remote endpoint is still there. This is achieved via the server parameters ClientAliveInterval/ClientAliveCountMax and corresponding client ServerAliveInterval/ServerAliveCountMax parameters as documented in the man pages for ssh_config and sshd_config.`

`This protocol serves two purposes. It provides in-band data for a TCP connection in case there are firewalls or other intermediate devices that require traffic (over and above the presence of the TCP-level flag TCP_KEEPALIVE) to maintain state. The second purpose provides early termination of TCP connections whose connection state has either been discarded en-route or terminated due to end-point system crash and which would only then terminate after TCP connection timeouts (~2hr 20mins at default values).`

`Neither of these involves any determination of a user being idle.`
```